### ISSUE TITLE
Observation: Samples do not cleanup stale sessions correctly

### ISSUE THREAD

**Issue by suggestedfixes** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#442]
From reading sample source, we understand that sessions are intended to be cleaned up within 5 seconds after termination. However, we did an experiment printing number of sessions count in the cleanup loop, here are the log entries. So it ended up not cleaning up the number of sessions after multiples of 5 seconds. To replicate, open a session with aws console, then close after connection.

```
2020-05-10 20:58:54 DEBUG   freeSampleStreamingSession(): Freeing streaming session with peer id: k497gf2c0rntc5zuop5pjq                                                                                                                     2020-05-10 20:58:54 DEBUG   socketConnectionClosed(): Close socket 22
2020-05-10 20:58:54 DEBUG   socketConnectionClosed(): Close socket 21
2020-05-10 20:58:55 DEBUG   socketConnectionClosed(): Close socket 23                                                                                                                                                                        2020-05-10 20:59:01 DEBUG   lwsWssCallbackRoutine(): Client is writable                                                                                                                                                                      2020-05-10 20:59:02 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 20:59:07 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 20:59:11 DEBUG   lwsWssCallbackRoutine(): Client is writable
2020-05-10 20:59:12 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 20:59:17 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 20:59:22 DEBUG   lwsWssCallbackRoutine(): Client is writable
2020-05-10 20:59:22 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 20:59:27 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 20:59:32 DEBUG   lwsWssCallbackRoutine(): Client is writable
2020-05-10 20:59:32 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 20:59:37 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 20:59:42 DEBUG   lwsWssCallbackRoutine(): Client is writable
2020-05-10 20:59:42 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 20:59:47 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 20:59:52 DEBUG   lwsWssCallbackRoutine(): Client is writable
2020-05-10 20:59:52 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 20:59:57 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 21:00:02 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 21:00:02 DEBUG   lwsWssCallbackRoutine(): Client is writable
2020-05-10 21:00:07 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 21:00:12 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 21:00:12 DEBUG   lwsWssCallbackRoutine(): Client is writable
2020-05-10 21:00:17 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 21:00:22 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 21:00:22 DEBUG   lwsWssCallbackRoutine(): Client is writable
2020-05-10 21:00:27 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 21:00:32 DEBUG   sessionCleanupWait(): >>>Number of sessions 1
2020-05-10 21:00:32 DEBUG   lwsWssCallbackRoutine(): Client is writable
2020-05-10 21:00:37 DEBUG   sessionCleanupWait(): >>>Number of sessions 1                
```

**Comment by suggestedfixes** [comment]
Here is where is LOGD is added
```
https://github.com/awslabs/amazon-kinesis-video-streams-webrtc-sdk-c/blob/master/samples/Common.c#L768
```

**Comment by suggestedfixes** [comment]
Should have added to 786, but still we should expect it to cleanup under 2 cycles ~10 seconds. Also the broadcast signal after freeing the session didn't seem to do anything.

**Comment by chehefen** [comment]
@suggestedfixes is this still an issue now after the DTLS close change?

**Comment by chehefen** [comment]
closing as the issue should be fixed by #481
